### PR TITLE
Add Oldsmobile models

### PR DESCRIPTION
**Pull request by prog1dev**
Resolves https://github.com/Bjorn248/scca_classifier/issues/58

### CODE REVIEW

File: common.js [hunk 1]
```diff
@@ -1708,6 +1708,63 @@ const allSoloCars = {
...
+      'all': ['hs'],
+    },
+    'Starfire (V6)': {
+      'all': ['esp'],
+    },
+    'Sunbird (V6)': {
+      'all': ['esp'],
+    },
+    'Omega': {
+      'all': ['f'],
```
Comment (Bjorn248) [inline common.js:1722]: This is not a valid class. Did you mean fsp?

File: common.js [hunk 2]
```diff
@@ -1708,6 +1708,63 @@ const allSoloCars = {
...
     },
   },
+  'Oldsmobile': {
+    'Calais W41': {
+      'all': ['hs'],
+    },
+    'Starfire (V6)': {
+      'all': ['esp'],
+    },
+    'Sunbird (V6)': {
```
Comment (Bjorn248) [inline common.js:1718]: This is not an Oldsmobile model

File: common.js [hunk 3]
```diff
@@ -1708,6 +1708,63 @@ const allSoloCars = {
...
+      'all': ['esp'],
+    },
+    'Omega': {
+      'all': ['f'],
+    },
+    'El Camino': {
```
Comment (Bjorn248) [inline common.js:1724]: This is not an Oldsmobile model

**Review by Bjorn248 — CHANGES_REQUESTED** [review]
Looks good other than a few invalid models and typos

File: common.js (hunk 1 above)
Comment (prog1dev) [reply common.js]: yes

fixed it

**Pull request merged**